app/services/yank_rubygem_service.rb

```diff
@@ -0,0 +1,21 @@
+class YankRubygemService
```

Comment:
Is it worth it to introduce "service" folder (and pattern) for this small usecase? Wouldn't be enough to hide the logic in `User` model for example?